README.md

```diff
@@ -1,99 +1,29 @@
-
-CCF currently runs on [Intel SGX](https://software.intel.com/en-us/sgx)-enabled platforms. Because CCF uses the [Open Enclave SDK](https://github.com/openenclave/openenclave)
-as the foundation for running in an enclave, as Open Enclave supports new TEE technologies, CCF will be able to run on new platforms. Networks can be run on-premises,
-in one or many cloud-hosted data centers, including [Microsoft Azure](https://azure.microsoft.com/), or in any hybrid configuration.
-
-Ledger providers can use CCF to enable higher throughput and higher confidentiality guarantees for distributed ledger applications.
-CCF developers can write application logic (also known as smart contracts) and enforce application-level access control in several languages by conﬁguring CCF
-to embed one of several language runtimes on top of its key-value store. Clients then communicate with a running CCF service using HTTP requests over TLS.
+Leveraging the power of trusted execution environments (TEEs), decentralized systems concepts, and cryptography, CCF enables enterprise-ready multiparty computation or blockchains.
 
 ## Learn more and get started
 
- * Read the [CCF Technical Report](CCF-TECHNICAL-REPORT.pdf) for a more detailed description.
  * Browse the [CCF Documentation](https://microsoft.github.io/CCF/).
- * Explore the CCF open-source GitHub repo, which also contains application examples and sample scripts for provisioning and setting up confidential computing VMs using Azure.
+ * Read the [CCF Technical Report](CCF-TECHNICAL-REPORT.pdf) for a more detailed description.
  * Learn more about [Azure Confidential Computing](https://azure.microsoft.com/solutions/confidential-compute/) offerings like Azure DC-series (which support Intel SGX TEE)
    and [Open Enclave](https://github.com/openenclave/openenclave) that CCF can run on.
 
-## Getting Started on Azure Confidential Computing
-
-First, if you are checking out the CCF repository, run `git clone`:
-```bash
-git clone https://github.com/microsoft/CCF.git
-```
-
-Then, under `CCF/getting_started/`:
- * `create_vm/` contains scripts to create an ACC VM (`make_vm.sh`).
-   This script expects a valid Azure subscription name to be set, eg: `export SUBSCRIPTION=sub_name`.
- * `setup_vm/` contains ansible playbooks that need to be run on the VM once created, for it to be able to build CCF.
-   Running `./run.sh driver.yml ccf-dev.yml` will apply those playbooks to the VM.
-
-## Build and Test
-
-```bash
-cd CCF
-mkdir build
-cd build
-cmake -GNinja ..
-ninja
-```
-
-Run the tests.
+## Getting Started with CCF
 
-```bash
-cd build
-./tests.sh
-```
+* [Get started](https://microsoft.github.io/CCF/master/quickstart/index.html) with Azure confidentual compute and CCF. 
+* Learn how to [build](https://microsoft.github.io/CCF/master/quickstart/build.html) and [run](https://microsoft.github.io/CCF/master/quickstart/build.html#running-tests) a test network.
```

Comment:
I think this bullet point should go last and only mention running a test network, pointing to https://microsoft.github.io/CCF/master/users/deploy_app.html. 

I don't think we should point at how to build CCF (or if we do, in a different bullet point) but point at the releases instead (https://github.com/microsoft/CCF/releases).